lib/game_data.js

```diff
@@ -105,6 +105,14 @@ class GameData {
     */
     addAdditionalItemProperties(iteminfo) {
         if (!this.items_game || !this.items_game_cdn || !this.csgo_english) return;
+        
+        //get sticker codename/name
+        for(var i=0;i<iteminfo.stickers.length;i++){
```

Comment:
We can use the `of` operator here.

Ex. `for (let sticker of iteminfo.stickers)`